.env

```diff
@@ -7,20 +7,20 @@
 
 ### DO NOT CHANGE THIS VALUE !!
 ### It should be updated only when you pull latest changes off from the 'master' branch of IntelOwl.
-INTELOWL_TAG_VERSION=v1.9.0
+INTELOWL_TAG_VERSION=v1.9.1
```

Comment:
why is this part of this PR ? the version number is `v1.9.1` in `develop` branch already.

Reply:
I pulled the changes. Hence🤷‍♂️

Reply:
a `git` rebase would have worked better I think

Reply:
Actually, I did a mistake in the start. I should have created another branch and then keep rebasing from develop. 😒. Hurriedly, I worked on develop branch and so, pulls are reflecting directly here.


Sorry, I'll create branches now on.

Reply:
Should I make any changes now?